docker-compose.yml

```diff
@@ -31,13 +31,15 @@ services:
     volumes:
     - .:/openemr:ro
     - .:/var/www/localhost/htdocs/openemr:rw
-    - assetsvolume01:/var/www/localhost/htdocs/openemr/public:rw
```

Comment:
After having a better understanding of this issue, it may be possible to remove all of the shared volumes (except for log), and then just add one shared volume called:
trickstervolume:/var/www/localhost/htdocs/openemr:rw

Reply:
Need to confirm first that a change in the host will show up in the docker (after the docker is up) in one of the shared drive directories before doing this.

Reply:
nope, it didn't work. the problem is that the directories/files we set as shared volumes will not allow changes from the host to the docker. so we need to be as granular as possible.